config/config/safety_center.yml

```diff
@@ -0,0 +1,150 @@
+statuses:

```

Comment:
We declare these statuses in a config file, but we also have code that depends on these literal names.  Assuming that YAML files support comments, can we add one here to say that?  I assume that deployments could add to this list without harm, but if they remove something or change names (e.g. to non-English ones), things will break, right?